### ISSUE TITLE
Kusto Notebook: on restart, connection information disappears, need to recreate a connection

### ISSUE THREAD

**Issue by baselsalam** [microsoft/azuredatastudio#12566]
- Azure Data Studio Version: 1.22.0 (user setup)
Commit: 77b9a708df3679c26a89ba24a070b3f7a078a510
Date: 2020-09-21T20:23:13.120Z
VS Code: 1.49.0
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.17763

Steps to Reproduce:

1. File > New Notebook.
2. + Code
3. Type a kusto query
4. Choose Kernel: 'Kusto' and Create a new connection.
5. Click on blue run arrow to run query
6. Restart Kusto Explorer.

The notebook opens fine, the query loads along with last results, but the connection is gone.

**Comment by MsSQLGirl** [comment]
The current default behavior of SQL or Kusto notebook today is that connection details in notebook is not persisted. 
This is a good feature request. 
@chlafreniere - to confirm / chime in.

**Comment by baselsalam** [comment]
Still repro's in v0.1.1 of the extension.

**Comment by chlafreniere** [comment]
Yes, this is currently expected behavior. We're currently working on persisting this connection information so that users don't need to re-select a connection every time (such as in this scenario)
